config/application.rb

```diff
@@ -64,6 +64,12 @@ class DavidRunger::Application < Rails::Application
   config.active_record.default_timezone = :utc
 
   config.middleware.insert_after(ActionDispatch::Static, Rack::Deflater) # gzip all responses
+  initializer(
+    'move Browser middleware after Rack::Attack',
+    after: 'rack-attack.middleware',
+  ) do |app|
+    app.middleware.move_after(Rack::Attack, Browser::Middleware)
+  end
```

Comment:
(unrelated change; I wanted to bring all of the middleware-related config together)